app/components/trainees/record_actions/view.html.erb

```diff
@@ -2,7 +2,11 @@
   <div class="record-actions">
 
     <% if can_recommend_for_award? %>
-      <%= render GovukButtonLinkTo::View.new(body: t("views.trainees.edit.recommend_for_award"), url: edit_trainee_outcome_details_outcome_date_path(trainee), class_option: "govuk-!-margin-bottom-0") %>
+      <% if trainee.is_early_years? %>
+        <%= render GovukButtonLinkTo::View.new(body: t("views.trainees.edit.recommend_for_eyts"), url: edit_trainee_outcome_details_outcome_date_path(trainee), class_option: "govuk-!-margin-bottom-0") %>
+      <% else %>
+        <%= render GovukButtonLinkTo::View.new(body: t("views.trainees.edit.recommend_for_award"), url: edit_trainee_outcome_details_outcome_date_path(trainee), class_option: "govuk-!-margin-bottom-0") %>
+      <% end %>
```

Comment:
Instead of repeating the full render line. it'll be better if you create a method on `view.rb` for body content, and you won't need to do the if statement here.